openupgrade_scripts/scripts/product/18.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,109 @@
+---Models in module 'product'---
+new model product.combo
+new model product.combo.item
```

Comment:
add a check in pre-migration, like:

`if openupgrade.table_exists(env.cr, "pos.combo"):`

then, add the model and table rename for both (`pos.combo` -> `product.combo` and `pos.combo.line` -> `product.combo.item`).

A part from doing this, I don't think anything else is necessary, as new fields are new features, computes or related. You can check analysis file if you do a rebase.

Reply:
I intended to do this when doing looking into point_of_sale, but now looked at those models anyways. We have to rename combo_price to extra_price in addition to the above.

Difficult to test though as it is now.